_posts/elements/2015-03-25-headings.md

```diff
@@ -20,78 +20,78 @@ info: Headings are used to define important topics and section content into dige
 
-    <h6 class="hd-6 emphasized">The Emerald Servant</h6>
-    <p class="copy copy-meta">Much and mandrill hey and guinea elephant debonair and before under complete in hippopotamus shot human precisely the the and uneasy as beside darn less overdid much far unblushingly drew favorable and limp yikes and dear cold and sentimentally teasing that since moronically oh eagle knew boyish pangolin.</p>
+    <h6 class="hd-6 emphasized">The Emerald Servant (emphasized h6)</h6>
+    <p class="copy copy-meta">Much and mandrill hey and guinea elephant debonair and before under complete in hippopotamus shot human precisely the the and uneasy as beside darn less overdid much far unblushingly drew favorable and limp yikes and dear cold and sentimentally teasing that since moronically oh eagle knew boyish pangolin. (meta copy)</p>
 
-    <h6 class="hd-6 de-emphasized">Destruction in the Storms</h6>
-    <p class="copy copy-meta">Instead this well dwelled to combed through less immeasurably the that flagrant jeez fanatically astride and treacherous outrageous much fish more antagonistic a toward charming dragonfly noiselessly far irrespective laboriously parrot beyond much squid well some that annoyingly severe but much slapped but liberally diverse.</p>
+    <h6 class="hd-6 de-emphasized">Destruction in the Storms (de-emphasized h6)</h6>
+    <p class="copy copy-meta">Instead this well dwelled to combed through less immeasurably the that flagrant jeez fanatically astride and treacherous outrageous much fish more antagonistic a toward charming dragonfly noiselessly far irrespective laboriously parrot beyond much squid well some that annoyingly severe but much slapped but liberally diverse. (meta copy)</p>
 </div>
 
 <div class="example-set">
-    <h6 class="hd-7">Slaves of Birch</h6>
-    <p class="copy copy-micro">Sourly browbeat yikes cantankerous jeepers far gosh crucially darn that so this the <a href="#">grumbled where jeez</a> and bid around the by warthog swung pending this stared hired and breezy stank bald a as some however dear since more more.</p>
+    <h6 class="hd-7">Slaves of Birch (h7)</h6>
+    <p class="copy copy-micro">Sourly browbeat yikes cantankerous jeepers far gosh crucially darn that so this the <a href="#">grumbled where jeez</a> and bid around the by warthog swung pending this stared hired and breezy stank bald a as some however dear since more more. (micro copy)</p>
 
-    <h5 class="hd-7 emphasized">The Heat's Game</h5>
-    <p class="copy copy-micro">Next stupid and on stuffily bald because domestically grew factious sewed outgrew coughed hello this overcame violent briefly much more improperly prior more experimental jeepers that egotistically licentiously unlike while darn regarding slowly mandrill until this dear on that mischievous before that insincere robin in nervelessly and.</p>
+    <h5 class="hd-7 emphasized">The Heat's Game (emphasized h7)</h5>
```

Comment:
Q: Should this (and the next) heading be an h6 instead of and h5 or h4? I wasn't sure if the change in headings used in  the html was intended to show how you can apply the h7 styling to multiple levels of HTML elements (h4 --> h6) or if this was a typo.